tests/common/fixtures/services.py

```diff
@@ -0,0 +1,25 @@
+from unittest.mock import create_autospec
+
+import pytest
+
+from h.services.search_index import SearchIndexService
+
+# Allow people to mass import everything without getting things we don't
+# mean them to like pytest.
+__all__ = ("service_mocker", "search_index")
```

Comment:
```suggestion
__all__ = ("service_mocker", "search_index")
```

The comment may be unnecessary, since I think `import *` is the only real reason to use an `__all__` anyway.

But why not just put these in a `tests/conftest.py` file? Then these fixtures would automatically be available to all tests (unit and functional) without the other `conftest.py` files having to import them